### ISSUE TITLE
Does not work `error` function in a loaded file by using `dofile`.

### ISSUE THREAD

**Issue by kohkimakimoto** [yuin/gopher-lua#67]
I tested the following 2 files code by using `glua`.

a.lua

```lua
dofile "b.lua"
```

b.lua

```lua
error("error!")
```

Run the command. Got a incorrect message: `nil` as the following.

```
$ glua a.lua
nil
stack traceback:
	[G]: in error
	b.lua:1: in <b.lua:0>
	[G]: in dofile
	a.lua:1: in function 'main chunk'
	[G]: ?
```

And I also tested running the original C lua5.1 in the same way. Got a correct error message.

```
$ lua5.1 a.lua
lua5.1: b.lua:1: error!
stack traceback:
	[C]: in function 'error'
	b.lua:1: in main chunk
	[C]: in function 'dofile'
	a.lua:1: in main chunk
	[C]: ?
```

**Comment by yuin** [comment]
Thanks for you PR.

But it seems just `dofile` bug. Lua reference manual says "that is, dofile does not run in protected mode" .
So I've changed `PCall` to `Call` in `dofile` .

**Comment by yuin** [comment]
I've fixed the issue in cfea889ae9364dcec3ba7b6ab38a07d2c792faed, and add a comment on #68 .
